### PR TITLE
Feat/511 remove derivation nodes

### PR DESCRIPTION
**Pull request by Sm00g15**

### CODE REVIEW

**Comment by umurb** [comment]
Please do not allow Default Derivation Path to be removed.
Can we extend the remove feature to Nodes as well (again Cryptonomic nodes not to be removed)

File: app/containers/HomeSettings/index.js [hunk 1]
```diff
@@ -173,6 +190,63 @@ class SettingsPage extends Component<Props> {
...
+        } else {
+          await this.handlePathChange('Default');
+        }
+      } else {
+        removePath(label);
+      }
+    }
+  };
+
+  removeNode = async (event, name) => {
```
Comment (developer0623) [inline app/containers/HomeSettings/index.js]: There are some issues on this function.
if (JSON.parse(localStorage.settings).nodesList.length > 4) {
      await this.handleConseilChange('Cryptonomic-Conseil');
} else {
       this.handleConseilChange('Cryptonomic-Conseil');
 }

File: app/containers/HomeSettings/index.js [hunk 2]
```diff
@@ -511,6 +595,8 @@ function mapDispatchToProps(dispatch) {
       removeNode,
       setLocale,
       setPath,
+      removePath,
+      updateNode,
```
Comment (developer0623) [inline app/containers/HomeSettings/index.js]: Where did you use `updateNode`?

Comment (Sm00g15) [reply app/containers/HomeSettings/index.js]: it was superfluous

File: app/containers/HomeSettings/index.js (hunk 1 above)
Comment (Sm00g15) [reply app/containers/HomeSettings/index.js]: ah yes, good catch I've cleaned that function up now.

**Pull request merged**